src/py/aspen/database/models/accessions.py

```diff
@@ -0,0 +1,87 @@
+import enum
+from typing import TYPE_CHECKING
+
+import enumtables
+from sqlalchemy import (
+    Column,
+    event,
+    ForeignKey,
+    Integer,
+    String,
+    text,
+    UniqueConstraint,
+)
+from sqlalchemy.orm import backref, relationship
+
+from .base import base, idbase
+from .entity import Entity
+from .enum import Enum
+
+
+class PublicRepositoryType(enum.Enum):
+    GISAID = "GISAID"
+    NCBI_SRA = "NCBI_SRA"
+
+
+# Create the enumeration table
+# Pass your enum class and the SQLAlchemy declarative base to enumtables.EnumTable
+_PublicRepositoryTypeTable = enumtables.EnumTable(
+    PublicRepositoryType,
+    base,
+    tablename="publicrepositorytypes",
```

Comment:
What's the naming convention? I would have guessed this would be `public_repository_types`. Are we doing something special for the enum tables with the names?

Reply:
I can change it.